### PR TITLE
Issue 212

### PR DESCRIPTION
**Pull request by prb112**

### CODE REVIEW

File: docs/FHIRServerUsersGuide.md [hunk 1]
```diff
@@ -1167,87 +1167,6 @@ and then we could modify the user-defined schematron rule as follows:
 
 By doing this, we have a much more flexible user-defined validation mechanism that allows us to apply rules only when 'http://ibm.com/fhir/profile/partner' is present in the instance.
 
-## 4.5 Encryption and decryption of requests and responses
```
Comment (lmsurpre) [inline docs/FHIRServerUsersGuide.md]: The sucky thing about our current markdown format is that when you remove a section, you need to update the section numbers of all the sections that follow it.  Sorry :-(

Comment (lmsurpre) [reply docs/FHIRServerUsersGuide.md]: And the index at the top of the document :-/

File: docs/FHIRServerUsersGuide.md [hunk 2]
```diff
@@ -1167,87 +1167,6 @@ and then we could modify the user-defined schematron rule as follows:
...
-
-### 4.5.4 Response processing
-On the response path, if the `Accept-Encoding` header is set in the request and contains the string `aescbc256`, the servlet filter encrypts the response body using the same encryption algorithm, configured encryption key and a randomly-generated IV. The IV will be specified in the `AES-Salt` response header.
-
-The encrypted response data is then transmitted by the servlet container back to the API consumer, which decrypts the response body (perhaps via a JAX-RS ReaderInterceptor implementation), using the shared encryption key and the IV obtained from the `AES-Salt` header found in the HTTP response.
-
-### 4.5.5 Configuration
-The configuration of the encryption feature consists of the generation of an encryption key, plus the definition of a few properties in the `fhir-server-config.json` file.
```
Comment (lmsurpre) [inline docs/FHIRServerUsersGuide.md]: we should find and remove the associated properties in 
1. the java properties-handling constants/code
2. any fhir-server-config samples that have it
3. section 5 of this document (if they are there)

File: docs/FHIRServerUsersGuide.md [hunk 3]
```diff
@@ -1167,87 +1167,6 @@ and then we could modify the user-defined schematron rule as follows:
...
-
-In addition to generating the encryption key and storing it within the keystore file, you also need to define a set of configuration properties like the following within the `fhir-server-config.json` file:
-
-```
-{
-    "fhirServer":{
-        ...
-        "encryption":{
-            "enabled":true,
-            "keystoreLocation":"resources/security/fhirkeys.jceks",
```
Comment (lmsurpre) [inline docs/FHIRServerUsersGuide.md]: we should remove this file now (from both the client and the server)

Comment (prb112) [reply docs/FHIRServerUsersGuide.md]: File is removed in the latest commit.

File: docs/FHIRServerUsersGuide.md (hunk 2 above)
Comment (prb112) [reply docs/FHIRServerUsersGuide.md]: updated 1/2/3

**Pull request merged**